notify/impl.go

```diff
@@ -135,6 +136,8 @@ func BuildReceiverIntegrations(nc *config.Receiver, tmpl *template.Template) []I
 
 const contentTypeJSON = "application/json"
 
+var userAgentHeader = fmt.Sprintf("Alertmanager/%s", version.Version)
```

Comment:
I tired that but I got this error

```
# github.com/prometheus/alertmanager/notify
notify/impl.go:139: const initializer fmt.Sprintf("Alertmanager/%s", version.Version) is not a constant
!! command failed: build -o /Users/st20638/.golang/src/github.com/prometheus/alertmanager/alertmanager -ldflags -X github.com/prometheus/alertmanager/vendor/github.com/prometheus/common/version.Version=0.7.1 -X github.com/prometheus/alertmanager/vendor/github.com/prometheus/common/version.Revision=e60ffbe6fb8c58f45864fccd1c741219402c2617 -X github.com/prometheus/alertmanager/vendor/github.com/prometheus/common/version.Branch=user-agent -X github.com/prometheus/alertmanager/vendor/github.com/prometheus/common/version.BuildUser=st20638@LF02Q331602A96M.local -X github.com/prometheus/alertmanager/vendor/github.com/prometheus/common/version.BuildDate=20170710-23:55:18  -s -a -tags netgo github.com/prometheus/alertmanager/cmd/alertmanager: exit status 2
make: *** [build] Error 1
```

I'm not familiar with Go, so I'm not sure how to fix that. When I tried this, it seems to work though

```
const (
	contentTypeJSON = "application/json"
	userAgentHeader = `Alertmanager/{{version.Version}}`
)
```

but again, I'm not familiar enough with Go to know if that's acceptable or not.